queue_services/entity-filer/src/entity_filer/filing_processors/continuation_out.py

```diff
@@ -0,0 +1,58 @@
+# Copyright © 2023 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""File processing rules and actions for the consent continuation out filing."""
+from contextlib import suppress
+from datetime import datetime
+from typing import Dict
+
+import dpath
+from legal_api.models import Business, Filing
+
+from entity_filer.filing_meta import FilingMeta
+from entity_filer.filing_processors.filing_components import filings
+
+
+def process(business: Business, continuation_out_filing: Filing, filing: Dict, filing_meta: FilingMeta):
+    """Render the continuation out filing into the business model objects."""
+    # update continuation out, if any is present
+    with suppress(IndexError, KeyError, TypeError):
+        court_order_json = dpath.util.get(filing, '/continuationOut/courtOrder')
+        filings.update_filing_court_order(continuation_out_filing, court_order_json)
+
+    continuation_out_json = filing['continuationOut']
+
+    details = continuation_out_json.get('details')
+    legal_name = continuation_out_json.get('legalName')
+    continuation_out_date = datetime.strptime(continuation_out_json.get('continuationOutDate'), '%Y-%m-%d')
+    foreign_jurisdiction = continuation_out_json.get('foreignJurisdiction')
+    foreign_jurisdiction_country = foreign_jurisdiction.get('country')
+
+    continuation_out_filing.order_details = details
+
+    if continuation_out_date >= datetime.now():
+        business.state = Business.State.HISTORICAL
+
+    business.jurisdiction = foreign_jurisdiction_country
+    business.foreign_legal_name = legal_name
+    business.continuation_out_date = continuation_out_date
+
+    with suppress(IndexError, KeyError, TypeError):
+        foreign_jurisdiction_region = foreign_jurisdiction.get('region')
+        business.foreign_jurisdiction_region = foreign_jurisdiction_region
+
+    filing_meta.continuation_out = {}
+    filing_meta.continuation_out = {**filing_meta.continuation_out,
+                                    **{'foreignJurisdictionCountry': foreign_jurisdiction_country},
+                                    **{'foreignJurisdictionRegion': foreign_jurisdiction_region},
+                                    **{'foreignLegalName': legal_name}}
```

Comment:
@argush3 I thought we can keep it as code in meta_data (since the UI submit codes to backend). I have done the same thing in consent continuation out. @severinbeauvais any objection?

Reply:
ok, yeah if FE is ok with this, then i'm good with using codes too.

Reply:
I can't remember if the UI receives country code for anything else ...?

But yes, we have a country code mixin/package that we can use to convert.

Reply:
@ketaki-deodhar @JazzarKarim FYI ^^ for whichever one of you implemented the ledger item

Reply:
ok, i'll cross that item out from the original comment then.

Reply:
@leodube-aot FYI ^^ in case YOU implement the ledger entry (tick 16253)

-> Ketaki and Karim could help you with it.